openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -400,102 +354,36 @@ def _show_login(self):
-            cmds.setAttr("rmanGlobals.imageOutputDir",
-                         "maya/<scene>/<layer>", type="string")
-
-    def _set_vray_settings(self, asset):
-        # type: (dict) -> None
-        """Sets important settings for Vray."""
-        settings = cmds.ls(type="VRaySettingsNode")
-        node = settings[0] if settings else cmds.createNode("VRaySettingsNode")
-
-        # set separator
-        # set it in vray menu
-        if cmds.optionMenuGrp("vrayRenderElementSeparator", exists=True,
-                              q=True):
-            items = cmds.optionMenuGrp(
-                "vrayRenderElementSeparator", ill=True, query=True)
-
-            separators = [cmds.menuItem(i, label=True, query=True) for i in items]  # noqa: E501
-            try:
-                sep_idx = separators.index(self.aov_separator)
-            except ValueError:
-                raise CreatorError(
-                    "AOV character {} not in {}".format(
-                        self.aov_separator, separators))
-
-            cmds.optionMenuGrp(
-                "vrayRenderElementSeparator", sl=sep_idx + 1, edit=True)
-        cmds.setAttr(
-            "{}.fileNameRenderElementSeparator".format(node),
-            self.aov_separator,
-            type="string"
-        )
-        # set format to exr
-        cmds.setAttr(
-            "{}.imageFormatStr".format(node), "exr", type="string")
-
-        # animType
-        cmds.setAttr(
-            "{}.animType".format(node), 1)
-
-        # resolution
-        cmds.setAttr(
-            "{}.width".format(node),
-            asset["data"].get("resolutionWidth"))
-        cmds.setAttr(
-            "{}.height".format(node),
-            asset["data"].get("resolutionHeight"))
-
-    @staticmethod
-    def _set_global_output_settings():
-        # enable animation
-        cmds.setAttr("defaultRenderGlobals.outFormatControl", 0)
-        cmds.setAttr("defaultRenderGlobals.animation", 1)
-        cmds.setAttr("defaultRenderGlobals.putFrameBeforeExt", 1)
-        cmds.setAttr("defaultRenderGlobals.extensionPadding", 4)
+        if "verify" not in kwargs:
+            kwargs["verify"] = not os.getenv("OPENPYPE_DONT_VERIFY_SSL", True)
+        return requests.post(*args, **kwargs)
+
+    def _requests_get(self, *args, **kwargs):
+        """Wrap request get method.
+
+        Disabling SSL certificate validation if ``DONT_VERIFY_SSL`` environment
+        variable is found. This is useful when Deadline or Muster server are
+        running with self-signed certificates and their certificate is not
+        added to trusted certificates on client machines.
+
+        Warning:
+            Disabling SSL certificate validation is defeating one line
+            of defense SSL is providing and it is not recommended.
+
+        """
+        if "verify" not in kwargs:
+            kwargs["verify"] = not os.getenv("OPENPYPE_DONT_VERIFY_SSL", True)
+        return requests.get(*args, **kwargs)
```

Comment:
Are theses methods unused?

Reply:
They are used in Muster code parts. But I would remove them along with the Muster specific code because Muster in its current for isn't working anyway and it all should be addressed in separate Issue (#3146). Also `_requests_get()` and `_requests_post()` should be library functions, probably used in place where specific requests calls are made (because studio might want to disable certificate check for globally, or just on specific endpoint)